packages/bvaughn-architecture-demo/components/sources/CommentButton.module.css

```diff
@@ -17,9 +17,11 @@
   display: flex;
   align-items: center;
   justify-content: center;
-  height: 2em;
-  min-width: 2em;
-  border-radius: 2em;
+  height: 2.5em;
+  min-width: 2.5em;
+  border-radius: 2.5em;
+  font-weight: 600;
+  font-family: "Inter";
```

Comment:
I've been trying to avoid a smattering of assorted font families in the new components– using the CSS font family variables instead. I think we should stick to that. I think consistency in fonts is good UI.

Reply:
There's a broader conversation to have here -- it's really good to start with a mathematically rigid approach, but the concept of "optical alignment" means that designs can often not feel right without some special-casing and fine-tuning.

Everyone's in agreement that we don't want to special-case everything because it looks amateur. But one of the most common tensions between design/dev is this topic -- trying to keep everything into a mathematical framework can lead to worse design, counter-intuitively.

So in this case, I'm surprised we need to set inter, but we absolutely need to be able to add more font weight sometimes.

Reply:
I did this because I didn't know where the variable was. By the way, this is for the add comment label, not the panel.

Reply:
FWIW we shouldn't use "Inter" (at least not by itself like this) – it's not a standard font and broke the button rendering for me.